src/App.js

```diff
@@ -15,59 +15,27 @@ function App() {
     }
   }, []);
   return (
-    <Container>
-      <AppHeader>
-        <AppLogo src={logo} alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <AppLink
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Your React {state.topic} Template
-        </AppLink>
-      </AppHeader>
-    </Container>
+    <Header>
+      <TopButtons />
+    </Header>
   );
 }
 
-const Container = styled.div`
-  text-align: center;
-`;
-
-const AppLogo = styled.img`
-  height: 40vmin;
-  pointer-events: none;
-
-  @media (prefers-reduced-motion: no-preference) {
-    animation: App-logo-spin infinite 20s linear;
-  }
-`;
-
-const AppHeader = styled.header`
-  background-color: #282c34;
-  min-height: 100vh;
+const Header = styled.div`
```

Comment:
Kleiner Verbesserungsvorschlag, um das html semantischer zu machen: 
```suggestion
const Header = styled.header`
```